android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/auth/InMemoryDAO.java

```diff
@@ -0,0 +1,77 @@
+package org.sagebionetworks.bridge.android.manager.auth;
+
+import android.content.Context;
+import android.support.annotation.AnyThread;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.sagebionetworks.bridge.rest.model.UserSessionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Created by jyliu on 1/20/2017.
+ */
+@AnyThread
+class InMemoryDAO implements AuthenticationManager.DAO {
```

Comment:
Is this class meant for unit testing?  I assume we will have another implementation for the app itself?